src/libraries/Common/src/System/CodeDom/CodeObject.cs

```diff
@@ -16,7 +16,7 @@ public class CodeObject
     internal class CodeObject
 #endif
     {
-        private IDictionary _userData;
+        private IDictionary? _userData;
```

Comment:
Is it OK to annotate files in the Common folder? Don't you need to add `#nullable enable` on top given that this file might be referenced by a project that doesn't have annotations enabled globally?

Reply:
As a reference: https://github.com/dotnet/runtime/pull/41438/files#r478296171

Reply:
The 3 files being annotated in the Common folder are only included in 1 other project: System.CodeDom, which only builds for netstandard2.0 and net461. On those TFMs the nullable warnings are disabled, so this should be fine. Also, the CI build would be failing if these were referenced by a project that doesn't have annotations enabled globally.